ic-canister/ic-canister-macros/src/lib.rs

```diff
@@ -19,6 +19,11 @@ pub fn canister_call(input: TokenStream) -> TokenStream {
     canister_call::canister_call(input)
 }
 
+#[proc_macro]
+pub fn canister_call_oneway(input: TokenStream) -> TokenStream {
```

Comment:
I suggest we call the macro `canister_notify`.

Reply:
Also, please, add documentation for new public macros in the crate. And also, to the crate level documentation of `ic-canister` crate.

Reply:
Using canister_notify is better for consistency with ic-cdk.
Before I thought its essence is oneway call, ic-cdk named notify is not a good idea. Anyway, let's use canister_notify.

Thanks for reminder about the document, I will refer to other interfaces to improve.